src/components/header/HeaderBody02.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Agissons from "../Agissons";
+import Footer from "../Footer";
+
+import {
+  Page,
+  BlockTitle,
+  ContainerTitle,
+  Title,
+  TitleSpan,
+  BlockSentence,
+  ContainerAuthorSentence,
+  AuthorSentence,
+  BlockButton,
+  ContainerMoreInformation,
+  MoreInformationSentence,
+} from "../../styled-components/header/HeaderBodyStyled02.jsx";
+
+export default function HeaderBody() {
+  const [visible, setVisible] = useState(false);
+  const onClick = () => {
+    setVisible(true);
+  };
+  return (
+    <>
+      <Page>
+        <BlockTitle>
+          <ContainerTitle>
+            <Title>
+              Comment nous <TitleSpan>agissons </TitleSpan>
+            </Title>
+          </ContainerTitle>
+        </BlockTitle>
+        <BlockSentence>
+          <ContainerAuthorSentence>
+            <AuthorSentence>
+              "Il faut être enthousiaste de son métier pour y exceller", Denis
+              Diderot
+            </AuthorSentence>
+          </ContainerAuthorSentence>
+        </BlockSentence>
+        <>
```

Comment:
Since this `<></>` tag won't have any incidence in your DOM, you shouldn't use it outside the classic "JSX wrapper" case

Reply:
(Repeat on similar files)